zeitschrift-fur-digitale-geisteswissenschaften.csl

```diff
@@ -0,0 +1,389 @@
+    </choose>
+  </macro>
+  <macro name="contributors-short">
+    <choose>
+      <if variable="author">
+        <names variable="author">
+          <name form="short" delimiter=" / " delimiter-precedes-last="always"/>
+        </names>
+      </if>
+      <else>
+        <choose>
+          <if type="book">
+            <names variable="editor">
+              <name form="short" delimiter=" / " delimiter-precedes-last="always"/>
+              <label prefix=" (" form="short" suffix=".)"/>
+            </names>
+          </if>
+        </choose>
+      </else>
+    </choose>
+  </macro>
+  <macro name="secondary-contributors">
+    <choose>
+      <if variable="author" type="book" match="all">
+        <names variable="editor" delimiter=" / ">
+          <label form="verb-short" text-case="lowercase" suffix=". "/>
+          <name delimiter=", " delimiter-precedes-last="always"/>
+        </names>
+      </if>
+    </choose>
+  </macro>
+  <macro name="container-contributors">
+    <choose>
+      <if variable="container-author">
+        <names variable="container-author">
+          <name delimiter=" / " delimiter-precedes-last="always"/>
+        </names>
+      </if>
+      <else>
+        <choose>
+          <if type="chapter">
+            <names variable="editor">
+              <label prefix=" " form="verb-short" suffix=" "/>
+              <name delimiter=" / " delimiter-precedes-last="always"/>
+            </names>
+          </if>
+        </choose>
+      </else>
+    </choose>
+  </macro>
+  <macro name="secondary-container-contributors">
+    <choose>
+      <if variable="container-author" type="chapter" match="all">
+        <names variable="editor">
+          <label form="verb-short" text-case="lowercase" suffix=". "/>
+          <name delimiter=", " delimiter-precedes-last="always"/>
+        </names>
+      </if>
+      <else-if type="article-journal article-magazine" match="any">
+        <names variable="editor">
+          <label form="verb-short" text-case="lowercase" suffix=". "/>
+          <name delimiter=" /" delimiter-precedes-last="always"/>
+        </names>
+      </else-if>
+    </choose>
+  </macro>
+  <macro name="secondary-collection-contributors">
+    <names variable="collection-editor">
+      <name delimiter=" / " delimiter-precedes-last="always"/>
+    </names>
+  </macro>
+  <macro name="translatedby">
+    <names variable="translator">
+      <label form="verb-short" text-case="lowercase" suffix=". "/>
+      <name delimiter=", " delimiter-precedes-last="always"/>
+    </names>
+  </macro>
+  <macro name="editor">
+    <names variable="editor">
+      <label form="verb-short" prefix=". " suffix=" "/>
+      <name delimiter=" / " delimiter-precedes-last="always"/>
+    </names>
+  </macro>
+  <macro name="title-long">
+    <group>
+      <choose>
+        <if type="book">
+          <text variable="title" form="long"/>
+          <text macro="editor"/>
+        </if>
+      </choose>
+      <choose>
```

Comment:
die nächsten vier lassen sich alle in eine `if` Kondition schreiben und das ganze am besten als `if` --`else-if` in ein einziges `choose` element

Reply:
Vielen Dank, habe ich umgesetzt